dags/telemetry_aggregates_parquet.py

```diff
@@ -0,0 +1,28 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'robhudson@mozilla.com',
+    'depends_on_past': True,
+    'start_date': datetime(2018, 9, 21),
+    'email': ['telemetry-alerts@mozilla.com', 'robhudson@mozilla.com', 'frank@mozilla.com'],
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 3,
+    'retry_delay': timedelta(minutes=30),
+}
+
+dag = DAG('telemetry_aggregates_parquet', default_args=default_args, schedule_interval='@daily')
+
+telemetry_aggregate_parquet_view = EMRSparkOperator(
+    task_id="telemetry_aggregate_parquet_view",
+    job_name="Telemetry Aggregate Parquet View",
+    instance_count=40,
```

Comment:
Does this really need 40 nodes?

Reply:
I copied what the telemetry aggregates needs. It does seem high but I don't know how to judge how many nodes is really needed.

Reply:
Those are the release aggregates. I would imagine this doesn't need more than 5 nodes, can you confirm?

Reply:
When I was testing it I was using a 5 node cluster on ATMO and less a few times and it ran fine.